make_dashboards/make_dashboards.py

```diff
@@ -47,7 +47,7 @@ def make_dashboards(outdir, demo, couch_user, password, couch_server, genstat_ur
     # Paths relative to make_dashboards/templates/
     internal_fn = os.path.join('internal','index.html')
     external_fn = os.path.join('external','index.html')
-    portal_fn = os.path.join('portal','index.html')
+    ngi_website_fn = os.path.join('ngi_website','index.html')
```

Comment:
@ewels Just a reminder we need to change the path in the cronjob to make this work

Reply:
Are you able to update the cronjob?